lib/babe/verify.go

```diff
@@ -409,15 +413,28 @@ func (b *verifier) verifyPreRuntimeDigest(digest *types.PreRuntimeDigest) (scale
 	case types.BabePrimaryPreDigest:
 		ok, err = b.verifyPrimarySlotWinner(d.AuthorityIndex, d.SlotNumber, d.VRFOutput, d.VRFProof)
 	case types.BabeSecondaryVRFPreDigest:
+		if !b.secondarySlots {
+			ok = false
+			break
+		}
 		pub := b.authorities[d.AuthorityIndex].Key
-		var pk *sr25519.PublicKey
-		pk, err = sr25519.NewPublicKey(pub.Encode())
+
+		pk, err := sr25519.NewPublicKey(pub.Encode()) // nolint
```

Comment:
Assuming this is because of govet's funny shadowing detection feature, use this instead:

```suggestion
		pk, err := sr25519.NewPublicKey(pub.Encode()) //nolint:govet
```

Reply:
I removed //nolint tag completely, and linter no longer complains, has something changed?

Reply:
Well yes, `golangci-lint` version wasn't pinned in the CI (see #1979), so maybe that was a false positive back months ago and now it doesn't detect it anymore.